docker_runner/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
 
 COPY ext/01_nodoc /etc/dpkg/dpkg.cfg.d/01_nodoc
 
 RUN apt-get -y update && \
     apt-get -y install --no-install-recommends\
         ca-certificates \
         locales \
-        python3-wheel \
-        python3-setuptools \
+        python3.8-venv \
```

Comment:
why did you change this to python3.8-venv, does not python3.8-wheel or setuptools exists?

Reply:
I had also a look, it seems so. The only thing you could do, install its dependencies individually, but this is overkill.
https://packages.ubuntu.com/focal/python3.8-venv